index.html

```diff
@@ -68,6 +68,177 @@ <h1 class="main__title">I'm Alexandra. <br> Glad to see you!</h1>
+            <figure class="work__image__container">
+                <img src="/assets/work2.png" alt="second work image" class="work__image">
+            </figure>
+            <div class="work__features__container">
+                <h2 class="work__title">Multi-Post Stories</h2>
+                <ul class="work__features">
+                    <li class="work__features__item">CANOPY</li>
+                    <li class="work__features__item">Back End Dev</li>
+                    <li class="work__features__item">2015</li>
+                </ul>
+                <p class="work__paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="work__list__container">
+                    <li class="work__language__item">html</li>
+                    <li class="work__language__item">css</li>
+                    <li class="work__language__item">javascript</li>
+                </ul>
+                <button class="work__button">See Project</button>
+            </div>
+        </article>
+        
+        <article class="work__description">
+            <figure class="work__image__container">
+                <img src="/assets/work3.png" alt="third work image" class="work__image">
+            </figure>
+            <div class="work__features__container">
+                <h2 class="work__title">Tonic</h2>
+                <ul class="work__features">
+                    <li class="work__features__item">CANOPY</li>
+                    <li class="work__features__item">Back End Dev</li>
+                    <li class="work__features__item">2015</li>
+                </ul>
+                <p class="work__paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="work__list__container">
+                    <li class="work__language__item">html</li>
+                    <li class="work__language__item">css</li>
+                    <li class="work__language__item">javascript</li>
+                </ul>
+                <button class="work__button">See Project</button>
+            </div>
+        </article>
+    </section>
+    <!-- END -->
+
+    <!-- ABOUT SECTION -->
+    <section id="about" class="about__section">
+        <article class="about__container">
+            <div class="about__information">
+                <h2 class="about__title">About Myself</h2>
+                <p class="about__paragraph">
+                    Hello I'm a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+                </p>
+                <span class="about__span">
+                    <a href="#" class="about__callToAction">LET'S CONNECT</a>
+                </span>
```

Comment:
[Optional] Well done on your implementation! I noticed that the text "LET'S CONNECT" is currently wrapped in a `<span>` tag. However, according to the design specifications, it should be considered as a subtitle for your social links.

To align your implementation with the design, I recommend updating the markup to use a more appropriate HTML element for a subtitle, such as a `<h2>` or `<h3>` tag. This will ensure that the structure and semantics of your document are consistent and clear.

Please make the necessary adjustment to use an appropriate HTML element for the subtitle.